### ISSUE TITLE
Content Item List search is case sensitive (Postgresql)

### ISSUE THREAD

**Issue by jptissot** [OrchardCMS/OrchardCore#5340]
The Content List search is case sensitive. This only seems to happen for Postgresql. YesSQL renders a `like ='search string'` where clause. For Postgresql, we would need to render the `ilike = ' search string'` for the search to be case insensitive

**Comment by jptissot** [comment]
@sebastienros do you have any ideas on how to fix this one ? Maybe a yessql issue instead ?

**Comment by deanmarcussen** [comment]
@jptissot yes this should probably be done in yessql, it should just be a matter of changing the dialect for postgresql (just a matter of making sure nothing else breaks from it...)

**Comment by sebastienros** [comment]
What about adding an extra column for searching on `DisplayText` that would be normalized: `NormalizedDisplayText`. And use it in this screen.

Note that we might have to do it for other fields that are used for search (TextField index) and also for queries that are used for unicity checks, like on Autoroute deduping.

**Comment by Skrypt** [comment]
I think we should use Lucene for that one and rely on the Lucene Analyzer for determining if it should be case sensitive or not. Lucene is made for that.

**Comment by jptissot** [comment]
PostgreSql 12 now supports case insensitive collations https://news.ycombinator.com/item?id=19991452

**Comment by Piedone** [comment]
So, it seems this isn't an issue with Postgres, right? And in the meantime we also have Lucene/Elasticsearch full-text search for the admin too.

**Comment by Piedone** [comment]
This actually is still an issue.

**Comment by Piedone** [comment]
Maybe it would be a solution to make YesSql have partial support for the second parameter of `Contains` and translate it to `ILIKE` when a case insensitive option is passed?
